examples/masks/main.rs

```diff
@@ -55,6 +55,12 @@ fn update(keyboard_input: Res<Input<KeyCode>>, mut query: Query<&mut Mask, With<
         mask.strength -= increment();
     };
 
+    if keyboard_input.pressed(KeyCode::PageUp) {
+        mask.fade += 0.01;
+    } else if keyboard_input.pressed(KeyCode::PageDown) {
+        mask.fade -= 0.01;
+    };
```

Comment:
After this block, could you clamp it? https://doc.rust-lang.org/std/primitive.f32.html#method.clamp

Mostly to show users clearly the valid range of `fade`, even though the shader will ensure it anyway.

Reply:
done

Reply:
no wait, github didnt add latest commit from my branch here. I thought it was supposed to

Reply:
would you prefer me to force push changes with the same commit or open another pr for this? or am i missing something and it can still be done in the same pr by me? the commit in question: https://github.com/gavlig/bevy-vfx-bag/commit/8fd66d7015223f28c167d678053dead9928b4492

Reply:
tried force pushing it all in a single commit still no luck :/ So if github doesnt let you fix this somehow on your side let me know and i'll make another pr

Reply:
No worries, I'll squash it into a single commit anyway. 
Thanks for the PR, the effect looks very nice.